nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -1760,9 +1766,13 @@ N/A
                            valuemask , &attributes);
       }
 
+      if (nxagentReportWindowIds)
+      {
+        fprintf (stderr, "NXAGENT_WINDOW_ID: SCREEN_WINDOW:%d,WID:[0x%x]\n", pScreen -> myNum, nxagentInputWindows[pScreen->myNum]);
```

Comment:
Hmm, why not enclose the screen number in brackets, too? That's usually how it is handled within all nxagent  debug/test outputs. Also normally we do not use ":". So I suggest:

```
fprintf (stderr, "NXAGENT_WINDOW_ID: SCREEN_WINDOW [%d][0x%x]\n", pScreen -> myNum, nxagentInputWindows[pScreen->myNum]);
```